docs/examples/aws-credentials-iam-role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
This isn't referenced from the examples page (or elsewhere?
Could you update the existing Aws example with some sommebts for optionality of the fields?

Reply:
I don't understand. Do you mean that I should add a reference within the examples README?

Reply:
Yes these docs/ will not be visible I documentation unless you reference them somewhere.
https://github.com/jenkinsci/kubernetes-credentials-provider-plugin/blob/master/docs/examples/README.md#aws-credentials